aten/src/ATen/Context.h

```diff
@@ -82,6 +82,9 @@ class TORCH_API Context {
   static bool hasHIP() {
     return detail::getHIPHooks().hasHIP();
   }
+  static bool hasHIPSOLVER() {
+    return detail::getHIPHooks().hasHIPSOLVER();
```

Comment:
I think (even though it looks weird) it will be more consistent with current implementation to call the `detail::getCUDAHooks().hasHIPSOLVER()` function here.